posts/host-your-own-cncf-cloudevents-compatible-event-gateway-kubernetes.md

```diff
@@ -0,0 +1,133 @@
+description: "The Serverless Event Gateway quickstart for Kubernetes. Host your own private Event Gateway, point to any FaaS provider with CloudEvents."
+date: 2018-08-16
+layout: Post
+thumbnail: https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/eg-kubernetes/event-gateway-kubernetes-thumb.png
+authors:
+  - SebastianBorza
+---
+
+<img src="https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/eg-kubernetes/event-gateway-kubernetes-header.png">
+
+Last year, we launched the [Event Gateway open source project](https://github.com/serverless/event-gateway): the event router for the serverless world.
+
+In a nutshell, the Event Gateway is the way you take your business events, and route them to serverless (FaaS) compute. You can use it to easily build APIs or react to custom business events. You can even connect events across different vendors to integrate your new applications with legacy infrastructure or other SaaS.
+
+While we do have a hosted (read: serverless) version of the Event Gateway contained inside our new [Serverless Platform](https://serverless.com/blog/serverless-platform-beta-helps-teams-operationalize-development/), the open source version is one that you have to host yourself.
+
+And today, doing that is about to get a lot easier. **You can now deploy your own self-hosted Event Gateway instance to a Kubernetes cluster in just a few commands!**
+
+Even better: the Event Gateway uses the CNCF’s [CloudEvents](https://github.com/cloudevents/spec) format. Meaning, it’s cloud-agnostic down to its core. Use it with any FaaS provider you want.
+
+Read on to learn how to get set up with the Event Gateway on your own Kubernetes cluster, or [read the full release notes in GitHub](NEED LINK).
+
+## Self-hosting the Event Gateway
+
+The Event Gateway is a distributed system. It’s designed to be highly-available and continue running even in the event that one of its component pieces fail.
+
+In order to self-host it, you would need to stand up all the underlying infrastructure for the Event Gateway. That means building out and managing the VM, setting up an etcd cluster, and handling ingress via a load balancer (nginx, proxy). This can be a complex undertaking.
+
+But now, if you’re already Kubernetes, self-hosting your own Event Gateway instance is easy. _YES._
+
+### Why Kubernetes?
+
+Kubernetes abstracts away all the support logic required in running your own infrastructure. You can just let the system do it for you. _YES x2._
+
+## How to run the Event Gateway right now, on Kubernetes
+
+Here are all the steps you need to get set up with a multi-node cluster.
+
+## Getting Started
+
+You'll need to have an existing Kubernetes cluster that supports Ingress for Deployments. The [instructions contained here](https://github.com/serverless/event-gateway/tree/master/contrib/MINIKUBE.md) outline how to get that configured with minikube's native nginx ingress.
```

Comment:
Shouldn't we link to the main README not to minikube guide? @sebito91 @alexdebrie